src/components/Modal/Modal.stories.tsx

```diff
@@ -67,6 +61,23 @@ Regular.args = {
     ],
 };
 
+export const VerticalCenter = Template.bind({});
```

Comment:
why is `isCentered` a prop? I would assume the Modal should always aim to be the the center of the screen horizontally and vertically by default. Is that not the case? Maybe you have some good evidence here perhaps? TY

Reply:
As I wrote in the PR description, I found in the design so of the modals to be not centred and some are centred 
Plus not centering it will help in the scrolling behaviour